templates/2_Pixie_Cluster_Pixels.ipynb

```diff
@@ -1,7 +1,6 @@
 {
```

Comment:
<p>Can we change this to: "You cannot use the same name for different meta clusters; doing so will cause the next step to fail."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/993/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>